test/padStart.js

```diff
@@ -0,0 +1,47 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('padStart', function() {
+  specify('should pad string with spaces', function() {
```

Comment:
specify -> it

Reply:
We only use `specify` inside `context`. In other cases `it` is used inside `describe`.